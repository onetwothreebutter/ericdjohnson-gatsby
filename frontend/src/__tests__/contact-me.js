import React from 'react'
import { shallow } from 'enzyme'
import shallowToJson from 'enzyme-to-json'

import ContactMePage from '../pages/contact-me'

test('Contact Me Page should render correctly', () => {
  const wrapper = shallow(<ContactMePage />)
  expect(shallowToJson(wrapper)).toMatchSnapshot()
})
