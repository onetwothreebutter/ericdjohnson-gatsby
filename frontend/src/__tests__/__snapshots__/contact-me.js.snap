// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact Me Page should render correctly 1`] = `
<div>
  <section
    className="resume__banner"
  >
    <div
      className="site__page-width"
    >
      <h1
        className="resume__banner-heading"
      >
        Resume & Contact Info
      </h1>
      <PhotoCredit
        image_url="http://unsplash.com/@helloimnik?utm_medium=referral&utm_campaign=photographer-credit&utm_content=creditBadge"
        name="Hello I'm Nik"
      />
    </div>
  </section>
  <section
    className="site__section"
  >
    <h2
      className="site__section-heading"
    >
      Resume
    </h2>
    <p
      className="resume__paragraph-text"
    >
      <object
        data={Object {}}
        style={
          Object {
            "display": "block",
            "height": "100%",
            "width": "100%",
          }
        }
        type="application/pdf"
      />
    </p>
  </section>
  <hr
    className="section-divider"
  />
  <section
    className="site__section"
  >
    <h2
      className="site__section-heading"
    >
      Contact Info
    </h2>
    <p
      className="contact__paragraph-text"
    >
      <a
        className="contact-link"
        href="mailto:eric.d.johnson@gmail.com"
      >
        <[object Object]
          alt="envelope with @ sign"
          className="email-icon"
          title="Icon made by https://www.flaticon.com/authors/ocha"
        />
        <span
          className="contact-link__text"
        >
          eric.d.johnson@gmail.com
        </span>
      </a>
    </p>
    <p
      className="contact__paragraph-text"
    >
      <a
        className="contact-link"
        href="tel:+13194009903"
      >
        <[object Object]
          alt="cell phone"
          className="phone-icon"
          title="Icon made by https://www.flaticon.com/authors/freepik"
        />
        <span
          className="contact-link__text"
        >
          (319) 400-9903
        </span>
      </a>
    </p>
    <p
      className="contact__paragraph-text"
    >
      <a
        className="contact-link"
        href="https://www.linkedin.com/in/er1cj0hns0n/"
      >
        <img
          alt="LinkedIn logo"
          className="linked-in-logo"
          src=""
        />
        <span
          className="contact-link__text"
        >
          linkedin.com/in/er1cj0hns0n
        </span>
      </a>
    </p>
  </section>
  <hr
    className="section-divider"
  />
</div>
`;
